django/forms/fields.py

```diff
@@ -532,8 +534,18 @@ class EmailField(CharField):
     widget = EmailInput
     default_validators = [validators.validate_email]
 
+    def __init__(self, *args, **kwargs):
+        strip = kwargs.pop('strip', None)
+        if strip is not None:
+            warnings.warn(
+                "Passing `strip` as keyword argument is deprecated. "
+                "EmailField values will always be stripped of leading and trailing whitespace.",
+                RemovedInDjango21Warning, stacklevel=2
+            )
+        super(EmailField, self).__init__(*args, strip=True, **kwargs)
+
     def clean(self, value):
-        value = self.to_python(value).strip()
+        value = self.to_python(value)
```

Comment:
With this change, can the entire `clean()` method be removed from `EmailField` and `URLField`? The parent implementation already [calls `to_python()`](https://github.com/django/django/blob/415ae960bb9f1bdae798023fdce3247d2c938eec/django/forms/fields.py#L158)?